gen/tests/test_adminrouter_tls_conf.py

```diff
@@ -12,15 +12,21 @@ class TestAdminRouterTLSConfig:
     Tests for the Admin Router TLS Config creation.
     """
 
-    def test_default(self):
+    @pytest.mark.parametrize(
+        'adminrouter_tls_1_0_enabled, tls_versions',
+        [('true', 'TLSv1 TLSv1.1 TLSv1.2'), ('false', 'TLSv1.1 TLSv1.2')]
+    )
+    def test_master(self, adminrouter_tls_1_0_enabled, tls_versions):
         """
         By default, the configuration specifies certain TLS settings.
```

Comment:
This docstring is now incorrect - this does not test what the default is with no extra arguments. That is now a missing test.